src/main/java/mate/controller/car/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,31 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/all")
```

Comment:
let's change this url to "/drivers/cars" or "/cars/current" to make it more informative. "/cars/all" looks like a link to a list of all cars.
and don't forget to change it in your jsp